app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -374,6 +384,8 @@ public void onCreate(Bundle savedInstanceState) {
 
     isHideToolbar = sharedPreferenceUtil.getPrefHideToolbar();
 
+    SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(this);
```

Comment:
We now use dependency injection for shared preferences check out #515. Its a bit complicated so take your time.

Reply:
Bump

Reply:
@mhutti1 I have made the changes. Please review.